src/coreclr/debug/runtimeinfo/datadescriptor.h

```diff
@@ -154,6 +154,34 @@ CDAC_TYPE_BEGIN(GCHandle)
 CDAC_TYPE_SIZE(sizeof(OBJECTHANDLE))
 CDAC_TYPE_END(GCHandle)
 
+// Metadata
+
+CDAC_TYPE_BEGIN(MethodTable)
+CDAC_TYPE_INDETERMINATE(MethodTable)
+CDAC_TYPE_FIELD(MethodTable, /*uint32*/, DwFlags, cdac_offsets<MethodTable>::m_dwFlags)
```

Comment:
Will rename the cdac_offsets specialization members.  I want to keep `DwFlags`/`DwFlags2` rather than just `Flags`/`Flags2` just because it's at least a little bit easier to search for.

Reply:
Nit: `dw` (DWORD) is ancient left-over from 16-bit Windows. It would be nice to avoid it and other Hungarian notation in the data contract.

If you want something that is easy to search for, you can make it `MTFlags` or something like that.